CMakeLists.txt

```diff
@@ -79,22 +80,28 @@ if(${COUNTER} GREATER 1)
 LEAKCHECK=${LEAKCHECK} \
 THREADCHECK=${THREADCHECK} \
 UNDEFINED_BEHAVIOR_CHECK=${UNDEFINED_BEHAVIOR_CHECK} \
-CODECOVERAGE=${CODECOVERAGE}")
+CODECOVERAGE=${CODECOVERAGE} \
+HEAPTRACK=${HEAPTRACK} \
+")
 endif()
 
 if(LEAKCHECK)
-    string(APPEND CMAKE_CXX_FLAGS " -fsanitize=leak -fsanitize=address")
+    string(APPEND CMAKE_CXX_FLAGS " -fsanitize=leak -fsanitize=address -DRUN_WITH_LEAKCHECK=1")
```

Comment:
It's more acceptable to add preprocessor definitions using `add_compile_definitions(RUN_WITH_LEAKCHECK)`

Reply:
Fixed, thanks!